sdk/sync/doc.go

```diff
@@ -1,12 +1,52 @@
-// Package sync contains the test instance synchronisation service.
+// The sync package contains the distributed coordination and choreography
+// facility of Testground. It is backed by Redis, and heavily inspired by
+// projects like Apache ZooKeeper, Apache Curator, etcd, and Consul.
```

Comment:
This comment would make me me wonder why it isn't just etcd, zookeeper, etc.

Reply:
I actually had a paragraph explaining why the choice, but I removed it as godocs are not a place for that. Good point. I will remove this, and move this kind of commentary to the user/dev docs instead.